Terminal.Gui/Views/TextView.cs

```diff
@@ -1962,7 +1962,25 @@ protected virtual void ColorNormal (List<Rune> line, int idx)
 		/// <param name="idx"></param>
 		protected virtual void ColorSelection (List<Rune> line, int idx)
 		{
-			Driver.SetAttribute (ColorScheme.Focus);
+			Driver.SetAttribute (new Attribute (ColorScheme.Focus.Background, ColorScheme.Focus.Foreground));
+		}
+
+		/// <summary>
+		/// Sets the <see cref="View.Driver"/> to an appropriate color for rendering the given <paramref name="idx"/> of the
+		/// current <paramref name="line"/>.  Override to provide custom coloring by calling <see cref="ConsoleDriver.SetAttribute(Attribute)"/>
+		/// Defaults to <see cref="ColorScheme.Focus"/>.
+		/// </summary>
+		/// <param name="line"></param>
+		/// <param name="idx"></param>
+		protected virtual void ColorReadOnly (List<Rune> line, int idx)
```

Comment:
Why does this take a `List<Rune> line` and `int idx` pram, if they are not used?

Reply:
I think @tznind only provide them for overridden them, see the `SyntaxHighlighting` scenario.

Reply:
So this method is declared on the `TextView` and is part of the render loop.  It is called for each index in the line.  It allows the user to alter the color set on the `Driver` based on arbitrary requirements (e.g. syntax highlighting).  

So for this reason it is not called `SetNormalColor` because it doesn't change the Normal color of the view.  It only changes the Color that is being used for that part of the current line as it is being rendered. 

I guess you could call it `SetDriverToNormalColor` though? 

It was implemented in this PR: https://github.com/gui-cs/Terminal.Gui/pull/1336

Reply:
I think the changes @tig requested and which I already did is enough. The explanation for the `line` and `idx` parameters not being used but handled on overridden methods are clarified.